homeassistant/components/weheat/manifest.json

```diff
@@ -6,5 +6,5 @@
   "dependencies": ["application_credentials"],
   "documentation": "https://www.home-assistant.io/integrations/weheat",
   "iot_class": "cloud_polling",
-  "requirements": ["weheat==2024.09.23"]
+  "requirements": ["weheat==2024.11.02"]
```

Comment:
This bump should be split out into its own separate PR.

Reply:
Why? The new release is specifically made for these changes; https://github.com/wefabricate/wh-python/compare/2024.09.23...2024.11.2

Reply:
Should I just create a single PR to bump this version?

Reply:
I do think so, it was asked of me as well: https://github.com/home-assistant/core/pull/126471 https://github.com/home-assistant/core/pull/125936